src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.java

```diff
@@ -68,6 +68,8 @@
 import com.google.gwt.user.client.ui.SimplePanel;
 import com.google.gwt.user.client.ui.Widget;
 
+import static org.rstudio.studio.client.rmarkdown.model.RmdChunkOutputUnit.TYPE_HTML;
```

Comment:
nit: feels awkward to have a static import of only this symbol and not the others; could we remove this and just qualify our usage?

Reply:
Oops, that was IntelliJ "helping". I've removed it.